test/test.py

```diff
@@ -0,0 +1,78 @@
+"""Tests
+
+How to run: python -m unittest discover
+"""
+import os
+import sys
+import unittest
+from pathlib import Path
+
+import pandas as pd
+
+TEST_DIR = os.path.dirname(__file__)
+PROJECT_ROOT = Path(TEST_DIR).parent
+sys.path.insert(0, str(PROJECT_ROOT))
+
+TEST_INPUT_DIR = os.path.join(PROJECT_ROOT, 'output')
+TEST_INPUT_RELEAES_DIR = os.path.join(TEST_INPUT_DIR, 'release')
+
+
+class TestGardMondoExactSssom(unittest.TestCase):
+
+    @classmethod
+    def setUpClass(cls):
+        """setUp"""
+        gard_mondo_exact_sssom_path = os.path.join(TEST_INPUT_RELEAES_DIR, 'gard-mondo-exact_curation.sssom.tsv')
+        cls.df = pd.read_csv(gard_mondo_exact_sssom_path, sep='\t', comment='#')
+
+    def test_dup_edges(self):
+        """Test that there are no duplicate edges"""
+        dup_edges = self.df[self.df.duplicated(subset=['subject_id', 'object_id'])]
+        self.assertEqual(len(dup_edges), 0, msg=f'Found {len(dup_edges)} duplicate edges')
+
+    # todo: This fails. I think not because of my code, but because of our mapping strategy. Need @Nico feedback
```

Comment:
Some new unit tests. This is the one that fails (duplicate mondo IDs in `gard-mondo-exact.sssom.tsv`)